test_team1.py

```diff
@@ -64,6 +64,30 @@ def test_twiter_handle(self):
         userName = 'powerade'
         self.assertFalse(find_twitter_handle(userName))
 
+    def test_bank_acc_number(self):
+        # Test valid bank account numbers
+        acc = '0123456789'
+        self.assertTrue(find_bank_acc_number(acc))
+
+        acc = '010234567891'
+        self.assertTrue(find_bank_acc_number(acc))
+
+        # Invalid Test
+        acc = '012.90.a89'
+        self.assertFalse(find_bank_acc_number(acc))
+
+    def test_email_address(self):
```

Comment:
It looks like you are missing an [import](https://github.com/jonettabaldwin/comp410_summer_2022/blob/ecfdf53ff11e04365d6e86873a687744cf728bd8/test_team1.py#L2) statement which causes your tests to fail